Resources/Textures/Objects/Power/Batteries/potato_battery.rsi/meta.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "https://github.com/tgstation/tgstation/",
+    "states": [
+        {
+            "name": "potato_battery",
+            "directions": 1,
+            "delays": [
+                [
+                    1.0
+                ]
```

Comment:
Not needed if there is no delay.

Reply:
This stuff gets put in there by the RSI editor (or rsi.py) and I don't like having to edit json files manually.